ensime-search.el

```diff
@@ -285,7 +285,7 @@
 	(goto-char target-point)
 	(setq ensime-search-selection-overlay
 	      (ensime-make-overlay target-point (point-at-eol)
-				   nil 'ensime-warnline-highlight))
+				   nil '(:face 'ensime-warnline-highlight)))
```

Comment:
might not have needed the `'` on `'ensime-warnline-highlight` but if it works then no big deal.